tools/harness-simulation/posix/etc/server

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Could you explain why we need this file? It seems almost identical to `ot-br-posix/script/server`. It contains some OTBR logic and may change over time. If we make a copy here we may need extra effort to maintain it.

Reply:
Compared to `ot-br-posix/script/server`, I only added two lines to start the service `commissionerd`. I have come up with other solutions to add the commands.
- One way is to use `diff` and `patch`. I think it is the most elegant way, but it will still behave incorrectly when there is a major revision over time.
- Only write the added commands to the file, and then use `cat >>` to append it to the original file. I think it will work on most cases, but the composed script may be a little bit weird in the final Docker image.

Thoughts?

Reply:
See https://github.com/openthread/ot-reference-release/blob/main/script/make-commissioner.bash.